frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.tsx

```diff
@@ -356,13 +431,24 @@ function getHasSettingsChanges({
 }
 
 function getNonDisabledEmbeddingParams(
-  embeddingParams: EmbeddingParameters,
-): EmbeddingParameters {
+  embeddingParams: EmbeddingParametersSettings,
+): EmbeddingParametersSettings {
   return Object.keys(embeddingParams).reduce((result, key) => {
     if (embeddingParams[key] !== "disabled") {
       result[key] = embeddingParams[key];
     }
 
     return result;
-  }, {} as EmbeddingParameters);
+  }, {} as EmbeddingParametersSettings);
+}
+
+function convertResourceParametersToEmbeddingParams(
```

Comment:
@oleggromov is this the same logic you implementend in your PR?
It seems to convert the missing parameters to "disabled"